--- package/feeds/packages/nano/Makefile	2021-11-26 13:36:46.951984313 +0000
+++ package/feeds/packages/nano/Makefile.orig	2021-11-26 12:32:42.639713890 +0000
@@ -31,7 +31,7 @@
   CATEGORY:=Utilities
   TITLE:=An enhanced clone of the Pico text editor
   URL:=https://www.nano-editor.org/
-  DEPENDS:=+libncurses
+  DEPENDS:=+libncurses +zlib +libmagic
 endef
 
 define Package/nano/description
@@ -40,9 +40,9 @@
 endef
 
 CONFIGURE_ARGS += \
-	--enable-tiny \
-	--disable-utf8 \
-	--disable-color \
+#	--enable-tiny \
+#	--disable-utf8 \
+#	--disable-color \
 	--enable-linenumbers
 
 CONFIGURE_VARS += \
